### DISCUSSION TITLE
Installation Challenge: srsRAN gNB with srsUE Project - Troubleshooting Open5gs Startup Error"

### DISCUSSION THREAD

**Discussion by patelsri** [srsran/srsRAN_Project#411] [General Help]
Hi everyone,

I am trying to install the srsRAN gNB with srsUE project. When I attempt to start the open5gs, I encounter an error:
### open5gs_5gc | 01/16 21:25:56.738: [upf] WARNING: PFCP[RSP] has already been associated [127.0.0.4]:8805

I have tried to edit the IP address of smf and upf in the open5gs-5gc.yml file, but it didn't work. Below, I have provided the full log of the Docker. Please check it."

aimslab@aimslab-ThinkPad-P16s-Gen-1:~/srsRAN_Project/docker$ sudo docker-compose up --build 5gc
[sudo] password for aimslab: 
WARNING: Some services (gnb) use the 'configs' key, which will be ignored. Compose does not support 'configs' configuration - use `docker stack deploy` to deploy to a swarm.
Building 5gc
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

Sending build context to Docker daemon   42.5kB
Step 1/20 : ARG OS_VERSION=22.04
Step 2/20 : FROM ubuntu:$OS_VERSION as base
 ---> 174c8c134b2a
Step 3/20 : ENV PYTHONBUFFERED=1
 ---> Using cache
 ---> 4f74eaea76c4
Step 4/20 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 01efeaa15a2a
Step 5/20 : RUN DEBIAN_FRONTEND=noninteractive apt-get update     && apt install -y software-properties-common     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> b9d6fcef2ada
Step 6/20 : RUN DEBIAN_FRONTEND=noninteractive apt-get update     && apt-get install -y     python3-pip     python3-setuptools     python3-wheel     ninja-build     build-essential     flex     bison     git     libsctp-dev     libgnutls28-dev     libgcrypt-dev     libssl-dev     libidn11-dev     libmongoc-dev     libbson-dev     libyaml-dev     libnghttp2-dev     libmicrohttpd-dev     libcurl4-gnutls-dev     libnghttp2-dev     libtins-dev     meson     curl     gettext     gdb     iproute2     iptables     iputils-ping     netcat     iperf     iperf3     libtalloc-dev     cmake     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 8ae9f68d18cc
Step 7/20 : ARG MONGO_MAJOR_VERSION=6
 ---> Using cache
 ---> a0c7b7f0aef9
Step 8/20 : RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y --no-install-recommends wget gnupg     && wget -qO - https://www.mongodb.org/static/pgp/server-${MONGO_MAJOR_VERSION}.0.asc | apt-key add     && . /etc/os-release     && echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu $UBUNTU_CODENAME/mongodb-org/${MONGO_MAJOR_VERSION}.0 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-${MONGO_MAJOR_VERSION}.0.list     && DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y --no-install-recommends mongodb-org     && apt-get autoremove && apt-get clean
 ---> Using cache
 ---> 8d45077d8586
Step 9/20 : ARG OPEN5GS_VERSION=v2.6.1
 ---> Using cache
 ---> 7c936b6fa80a
Step 10/20 : RUN echo $OPEN5GS_VERSION > ./open5gsversion
 ---> Using cache
 ---> 05be85a51e9b
Step 11/20 : RUN git clone --depth 1 --branch $(cat ./open5gsversion) https://github.com/open5gs/open5gs open5gs    && cd open5gs     && meson build --prefix=`pwd`/install     && ninja -C build     && cd build     && ninja install
 ---> Using cache
 ---> 44728e6d8f3e
Step 12/20 : ARG NODE_MAJOR=20
 ---> Using cache
 ---> 13203a80397b
Step 13/20 : RUN curl -fsSL https://deb.nodesource.com/setup_${NODE_MAJOR}.x | bash -     && apt-get install -y nodejs     && cd open5gs/webui     && npm ci --no-optional
 ---> Using cache
 ---> 7eb1c20596c2
Step 14/20 : RUN python3 -m pip install pymongo click pyroute2 ipaddress python-iptables
 ---> Using cache
 ---> 48f21b2945d3
Step 15/20 : FROM base as open5gs
 ---> 48f21b2945d3
Step 16/20 : WORKDIR /open5gs
 ---> Using cache
 ---> 90a35d71a40a
Step 17/20 : COPY open5gs-5gc.yml open5gs-5gc.yml.in
 ---> Using cache
 ---> 33f0f89358f6
Step 18/20 : COPY open5gs_entrypoint.sh add_users.py setup_tun.py subscriber_db.cs[v] ./
 ---> Using cache
 ---> 5f79d55e2283
Step 19/20 : ENV PATH="${PATH}:/open5gs/build/tests/app/"
 ---> Using cache
 ---> e507f4a1821a
Step 20/20 : ENTRYPOINT [ "./open5gs_entrypoint.sh" ]
 ---> Using cache
 ---> eb86d4ea45c1
Successfully built eb86d4ea45c1
Successfully tagged docker_5gc:latest
open5gs_5gc is up-to-date
Attaching to open5gs_5gc
open5gs_5gc       | nc: connect to 127.0.0.1 port 27017 (tcp) failed: Connection refused
open5gs_5gc       | waiting for mongodb
open5gs_5gc       | 
open5gs_5gc       | > open5gs@2.6.1 dev
open5gs_5gc       | > node server/index.js
open5gs_5gc       | 
open5gs_5gc       | > Using external babel configuration
open5gs_5gc       | > Location: "/open5gs/webui/.babelrc"
open5gs_5gc       | Connection to 127.0.0.1 27017 port [tcp/*] succeeded!
open5gs_5gc       |  DONE  Compiled successfully in 694ms9:25:52 PM
open5gs_5gc       | 
open5gs_5gc       | Mongoose: accounts.count({}, {})
open5gs_5gc       | > Ready on http://925a8ae98057:3000
open5gs_5gc       | Mongoose: accounts.findOne({ '$or': [ { username: 'admin' } ] }, { projection: { hash: 0, salt: 0 } })
open5gs_5gc       | Mongoose: subscribers.createIndex({ imsi: 1 }, { unique: true, background: true })
open5gs_5gc       | Mongoose: accounts.createIndex({ username: 1 }, { unique: true, background: true })
open5gs_5gc       | Mongoose: accounts.insertOne({ roles: [ 'admin' ], _id: new ObjectId("65a6f460cd9788776d2f37a2"), username: 'admin', salt: '19d948b3ea66196290ed5ef7580a03b3d80351176cfa00ce8fa56177b171c521', hash: '3e6b9dd7fa4e80194c3ece954f75333d2a29d1ddc8dd336168fdf33ae8c566d45d45e0af00a39ab502d94d87b1aca8b776d431edbded1b6404e423db96527e689eb28dc7ab0637a578956ea100b9eaa6d36e5b19a6aa59a89986b21714a6e27e3f6b70ded3d7bf63ace3ef15b120ff2a75b69f5dfdf971c8ee3b4770808e3a874fd3d7cf1e1a84979cd1fba036a811574ae7f9eb12510f14acc7258922e46117203978b3c7e963da5cfa8841a3ea95082ab1e1da28eedba713f2cc0b75b16837b71ce2da7ac969e40d7f00bd26fbd91fc510bdb64fcff74d3e5ff2a87251feac878d694af977220c64db79feef6b46998d62c70ee615246b0d49fc1533ac7d5282679438714baa31d9e4278cd499ac5d9bad5156090689475610047e91c1e789b15c753403bbaf30ad36f0e11889670a118e897221c5eabdf7613c60533cce481e2c909add2948d2bfd413708672a3bb3081a96ae8e95c6468948d8d2fcd1d5b44eaa7b70982373fe933c80d025a6afd08b2c61654b2225746fda420ff2a95dda283e3102e6d30b36312255ed121ba280b3d82c09fa6fa637a79c250e96a436fc96768d63ab94234d8a437665ee43b5f199758805ee69d3e5f841241bed117c24b088969076e1fec1a1d5665648f6c68411dd39f665531265f2bce4b1ae0237e50dfb0d57608278b570ba0f8ba43cfaad1c32aeaeeee3937f35264969958b9d9', __v: 0}, { session: null })
open5gs_5gc       | SUBSCRIBER_DB=001010123456780,00112233445566778899aabbccddeeff,opc,63bfa50ee6523365ff14c1f45f88737d,8000,9,10.45.1.2
open5gs_5gc       | Reading subscriber data from cmd.
open5gs_5gc       | Added subscriber with Inserted ID : 65a6f461ed4529990c657117
open5gs_5gc       | 65a6f461ed4529990c657117
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.124: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.127: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.128: [sbi] INFO: nghttp2_server() [http://127.0.0.10]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.129: [app] INFO: NRF initialize...done (../src/nrf/app.c:31)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.179: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.181: [sbi] INFO: nghttp2_server() [http://127.0.1.10]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.181: [app] INFO: SCP initialize...done (../src/scp/app.c:31)
open5gs_5gc       | 01/16 21:25:54.182: [nrf] INFO: [d47887f8-b4b5-41ee-b394-bbe6b94a4f3e] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.182: [sbi] INFO: [d47887f8-b4b5-41ee-b394-bbe6b94a4f3e] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.231: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.235: [pfcp] INFO: pfcp_server() [127.0.0.7]:8805 (../lib/pfcp/path.c:30)
open5gs_5gc       | 01/16 21:25:54.235: [pfcp] INFO: ogs_pfcp_connect() [127.0.0.4]:8805 (../lib/pfcp/path.c:61)
open5gs_5gc       | 01/16 21:25:54.235: [gtp] INFO: gtp_server() [10.53.1.2]:2152 (../lib/gtp/path.c:30)
open5gs_5gc       | 01/16 21:25:54.236: [app] INFO: UPF initialize...done (../src/upf/app.c:31)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.284: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.306: [gtp] INFO: gtp_server() [127.0.0.4]:2123 (../lib/gtp/path.c:30)
open5gs_5gc       | 01/16 21:25:54.306: [gtp] INFO: gtp_server() [127.0.0.4]:2152 (../lib/gtp/path.c:30)
open5gs_5gc       | 01/16 21:25:54.306: [pfcp] INFO: pfcp_server() [127.0.0.4]:8805 (../lib/pfcp/path.c:30)
open5gs_5gc       | 01/16 21:25:54.306: [pfcp] INFO: ogs_pfcp_connect() [127.0.0.7]:8805 (../lib/pfcp/path.c:61)
open5gs_5gc       | 01/16 21:25:54.306: [sbi] INFO: NF Service [nsmf-pdusession] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.306: [sbi] INFO: nghttp2_server() [http://127.0.0.4]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.306: [upf] INFO: PFCP associated [127.0.0.4]:8805 (../src/upf/pfcp-sm.c:173)
open5gs_5gc       | 01/16 21:25:54.306: [app] INFO: SMF initialize...done (../src/smf/app.c:31)
open5gs_5gc       | 01/16 21:25:54.307: [smf] INFO: PFCP associated [127.0.0.7]:8805 (../src/smf/pfcp-sm.c:174)
open5gs_5gc       | 01/16 21:25:54.307: [nrf] INFO: [d4890c22-b4b5-41ee-b20d-a7d450b5275a] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.307: [sbi] INFO: [d4890c22-b4b5-41ee-b20d-a7d450b5275a] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.307: [sbi] INFO: [d4890c22-b4b5-41ee-b20d-a7d450b5275a] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.307: [sbi] INFO: [d4890c22-b4b5-41ee-b20d-a7d450b5275a] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.334: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.337: [sbi] INFO: NF Service [namf-comm] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.338: [sbi] INFO: nghttp2_server() [http://127.0.0.5]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.338: [amf] INFO: ngap_server() [10.53.1.2]:38412 (../src/amf/ngap-sctp.c:61)
open5gs_5gc       | 01/16 21:25:54.338: [sctp] INFO: AMF initialize...done (../src/amf/app.c:33)
open5gs_5gc       | 01/16 21:25:54.338: [nrf] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.339: [sbi] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.339: [sbi] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.339: [sbi] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.339: [sbi] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.339: [sbi] INFO: [d490203e-b4b5-41ee-8ffc-89db26f43ee8] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.381: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.383: [sbi] INFO: NF Service [nausf-auth] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.383: [sbi] INFO: nghttp2_server() [http://127.0.0.11]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.383: [app] INFO: AUSF initialize...done (../src/ausf/app.c:31)
open5gs_5gc       | 01/16 21:25:54.383: [nrf] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.384: [sbi] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.384: [sbi] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.384: [sbi] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | 01/16 21:25:54.384: [sbi] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.384: [sbi] INFO: [d4974b98-b4b5-41ee-b196-5fb999d86922] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.433: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.434: [sbi] INFO: NF Service [nudm-ueau] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.434: [sbi] INFO: NF Service [nudm-uecm] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.434: [sbi] INFO: NF Service [nudm-sdm] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.435: [sbi] INFO: nghttp2_server() [http://127.0.0.12]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.435: [app] INFO: UDM initialize...done (../src/udm/app.c:31)
open5gs_5gc       | 01/16 21:25:54.435: [nrf] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] WARNING: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF has already been added (../lib/sbi/nnrf-handler.c:636)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] WARNING: NF EndPoint updated [127.0.0.12:80] (../lib/sbi/context.c:1618)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] WARNING: NF EndPoint updated [127.0.0.12:7777] (../lib/sbi/context.c:1527)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.436: [sbi] INFO: [d49f3934-b4b5-41ee-8021-6f94a440ecbb] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.488: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.491: [dbi] INFO: MongoDB URI: 'mongodb://127.0.0.1/open5gs' (../lib/dbi/ogs-mongoc.c:130)
open5gs_5gc       | 01/16 21:25:54.491: [sbi] INFO: NF Service [npcf-am-policy-control] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.491: [sbi] INFO: NF Service [npcf-smpolicycontrol] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.491: [sbi] INFO: NF Service [npcf-policyauthorization] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.491: [sbi] INFO: nghttp2_server() [http://127.0.0.13]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.491: [app] INFO: PCF initialize...done (../src/pcf/app.c:31)
open5gs_5gc       | 01/16 21:25:54.492: [nrf] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.492: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.493: [sbi] INFO: [d4a7a8da-b4b5-41ee-a7b4-6125f9d59b42] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.534: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.535: [sbi] INFO: NF Service [nnssf-nsselection] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.535: [sbi] INFO: nghttp2_server() [http://127.0.0.14]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.536: [app] INFO: NSSF initialize...done (../src/nssf/app.c:31)
open5gs_5gc       | 01/16 21:25:54.536: [nrf] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.536: [sbi] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.536: [sbi] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.537: [sbi] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.537: [sbi] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.537: [sbi] INFO: [d4ae9618-b4b5-41ee-9eaf-13013ee05caa] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.585: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.587: [sbi] INFO: NF Service [nbsf-management] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.587: [sbi] INFO: nghttp2_server() [http://127.0.0.15]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.587: [app] INFO: BSF initialize...done (../src/bsf/app.c:31)
open5gs_5gc       | 01/16 21:25:54.588: [nrf] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.588: [sbi] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.588: [sbi] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.588: [sbi] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.588: [sbi] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.588: [sbi] INFO: [d4b66e38-b4b5-41ee-824d-ed2ed3db1e31] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | Open5GS daemon v2.6.1
open5gs_5gc       | 
open5gs_5gc       | 01/16 21:25:54.644: [app] INFO: Configuration: 'open5gs-5gc.yml' (../lib/app/ogs-init.c:126)
open5gs_5gc       | 01/16 21:25:54.646: [dbi] INFO: MongoDB URI: 'mongodb://127.0.0.1/open5gs' (../lib/dbi/ogs-mongoc.c:130)
open5gs_5gc       | 01/16 21:25:54.646: [sbi] INFO: NF Service [nudr-dr] (../lib/sbi/context.c:1436)
open5gs_5gc       | 01/16 21:25:54.647: [sbi] INFO: nghttp2_server() [http://127.0.0.20]:7777 (../lib/sbi/nghttp2-server.c:395)
open5gs_5gc       | 01/16 21:25:54.647: [app] INFO: UDR initialize...done (../src/udr/app.c:31)
open5gs_5gc       | 01/16 21:25:54.647: [nrf] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] NF registered [Heartbeat:10s] (../src/nrf/nf-sm.c:190)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF registered (../lib/sbi/nnrf-handler.c:632)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] (NRF-notify) NF Profile updated (../lib/sbi/nnrf-handler.c:642)
open5gs_5gc       | 01/16 21:25:54.648: [sbi] INFO: [d4bf6bfa-b4b5-41ee-aded-b1ec48c25463] NF registered [Heartbeat:10s] (../lib/sbi/nf-sm.c:214)
open5gs_5gc       | 01/16 21:25:56.738: [smf] WARNING: PFCP[REQ] has already been associated [127.0.0.7]:8805 (../src/smf/pfcp-sm.c:217)
open5gs_5gc       | 01/16 21:25:56.738: [upf] WARNING: PFCP[RSP] has already been associated [127.0.0.4]:8805 (../src/upf/pfcp-sm.c:213)
open5gs_5gc       | 01/17 19:12:37.255: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.260: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.259: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.260: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.260: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.261: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.261: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.255: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.263: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.263: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)
open5gs_5gc       | 01/17 19:12:37.263: [core] ERROR: epoll failed (4:Interrupted system call) (../lib/core/ogs-epoll.c:216)

Thank You

**Comment by pgawlowicz** [comment]
Please try to use the new docker compose version 2. 
Specifically, 
`docker compose up --build 5gc` 
instead of the old version 1
`docker-compose up --build 5gc`
